kratos/utilities/auxiliar_model_part_utilities.h

```diff
@@ -31,12 +32,16 @@ namespace Kratos
 ///@{
 
     /// The index type definition
-    typedef std::size_t IndexType;
+    using IndexType = std::size_t;
```

Comment:
Please avoid defining a type in the Kratos namespace globally.

Reply:
Okay